src/components/input/input.scss

```diff
@@ -33,6 +39,15 @@ $md-input-underline-disabled-background-image: linear-gradient(to right,
   }
 }
 
+/**
+ * Applies a floating placeholder above the input before chrome's and safari's auto-fill commits.
+ * Firefox has selector issues for :-webkit-autofill, thats why this mixin is duplicated and is used seperately below 
+ */
+%md-input-placeholder-floating-autofill {
+  visibility: visible;
+  padding-bottom: 5px;
+  transform: translateY(-100%) scale(0.75);
+}
```

Comment:
You can just make this a mixin (rather than a placeholder) and use it in both places rather than duplicating it.